src/js/components/DataTable/README.md

```diff
@@ -396,6 +413,18 @@ Use this to indicate that 'data' doesn't contain all that it could.
 function
 ```
 
+**pin**
+
+Whether the header and/or footer should be pinned when
+      not all rows are visible. This only takes effect when 'fill' is true
```

Comment:
I could not find where is enforcement of the last sentence, I'll take another look in a few hours.

Reply:
removed